### PR TITLE
Updated Pat and Robbie

### PR DESCRIPTION
**Pull request by NeonAndrii**
Adding a new  Pat and a new Robbie

### CODE REVIEW

**Comment by NeonAndrii** [comment]
@NeonDaniel

**Comment by NeonDaniel** [comment]
@elon2 in case you wanted to look at the current scripts

File: pat_master.nct [hunk 1]
```diff
@@ -0,0 +1,379 @@
+Script: "Pat Test Andrii"
```
Comment (NeonDaniel) [inline pat_master.nct]: Script Title should match filename

File: robbie.nct [hunk 2]
```diff
@@ -0,0 +1,815 @@
+Script: "Covid 19 Audio"
```
Comment (NeonDaniel) [inline robbie.nct]: Script title should match filename

**Review by NeonDaniel — COMMENTED** [review]
Do these scripts replace existing ones? Old versions under different names should be removed if so.

**Comment by NeonAndrii** [comment]
I've added a new pat_master and robbie as separate (from pat_test_andrii and covid_19_audio) scripts so that we were able to check their performance and changes on our servers and only then get rid of the older versions.
@NeonDaniel would it be better (as an action template for the future) to replace them right away?

**Review by NeonDaniel — APPROVED** [review]

**Comment by NeonDaniel** [comment]
Great, thanks! I think it would be a good idea to automate naming at some point, we would just need to handle cases where 2 scripts have the same title, but are uploaded with different filenames. The only reason I wouldn't want to overwrite titles with filenames is that spaces, special characters, and capitalization could be lost.

**Pull request merged**